### ISSUE TITLE
[QUESTION] Should not found files not raise an error while executing collectstatic

### ISSUE THREAD

**Issue by Schwankenson** [jazzband/django-pipeline#776]
When executing `collectstatic` django-pipeline does not raise an error if a file defined in PIPELINE config is not found. Is this on purpose? Do I miss a settings? I would like to get an error if file which should be compiled does not exist.

**Comment by peymanslh** [comment]
Currently, Django-pipeline doesn't raise any error when files don't exist.

**Comment by Schwankenson** [comment]
@peymanslh Thank you, I know. The question is: Why does it not raise an error. I find it hard to debug my file locations without it throwing an error. Should it not raise an error, at least in debug mode?

**Comment by peymanslh** [comment]
The problem is pipeline doesn't work in debug mode, but it could be implemented as a feature with a flag in settings to throw an error when a file is not found. But as far as I know, it could be hard to implement!

**Comment by DUPLESSIS46** [comment]
band was born out of the stress of maintaining an Open Source project alone for a longer time. As such it’s aimed at projects with only a few maintainers that have not achieved creating a community of maintainers yet.

Jazzband strives to enable broader collaboration between the users by bringing down the walls that prevent them from becoming contributors.

Jazzband tries to foster participation by the community and shared responsibility within its membership. It helps to keep projects alive as well as finding new avenues of development.

In short, it’s cooperative coding.

**Comment by DUPLESSIS46** [comment]
Pip Tools
